buku.py

```diff
@@ -1740,11 +1738,165 @@ def exportdb(self, filepath, taglist=None):
         print('%s exported' % count)
         return True
 
+    def walk(self, root):
+        '''Recursively iterate over json
+
+        :param root: base node of the json data
+        :return: None
```

Comment:
No need to to add return if you are not explicitly returning something or `None`.